mod-source-record-storage-server/src/main/java/org/folio/verticle/consumers/DataImportConsumersVerticle.java

```diff
@@ -42,6 +43,7 @@ public class DataImportConsumersVerticle extends AbstractConsumerVerticle {
     DI_INVENTORY_AUTHORITY_CREATED_READY_FOR_POST_PROCESSING.value(),
     DI_INVENTORY_AUTHORITY_UPDATED_READY_FOR_POST_PROCESSING.value(),
     DI_INVENTORY_HOLDING_CREATED.value(),
+    DI_INVENTORY_HOLDING_UPDATED.value(),
```

Comment:
What about  DI_INVENTORY_ITEM_UPDATED? Do we need to add this subscription as well? For the case when Modify action is placed after the Update Item action

Reply:
It's already been added.  [   DI_INVENTORY_ITEM_UPDATED.value(),](https://github.com/folio-org/mod-source-record-storage/blob/77eda686feabf143d9922e170f14cea9895f1b29/mod-source-record-storage-server/src/main/java/org/folio/verticle/consumers/DataImportConsumersVerticle.java#L56)